### PR TITLE
New bid adapter for aniview

### PR DESCRIPTION
**Pull request by itaynave**
www.aniview.com
Please approve new bid adapter for aniview

## Type of change

- [ x] New bidder adapter   
- Done
## Description of change
New bid adapter for aniview

- test parameters for validating bids
```
{
  bidder: 'aniview',
  params: {
            AV_PUBLISHERID: '55b78633181f4603178b4568',
            AV_CHANNELID: '55b7904d181f46410f8b4568'
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x ] official adapter submission
itay@aniview.com
For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by idettman** [comment]
@itaynave could you provide me with an integration test page? Something like the `intergrationExamples/gpt/pbjs_video_adUnit.html`. I need to verify bids returned, but there is an external dependency `aniviewRenderer`, that I don't have in my test page so I am unable to test full functionality.

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/1135

File: modules/aniviewBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      s2sParams.AV_WIDTH = playerWidth;
+      s2sParams.AV_HEIGHT = playerHeight;
+      s2sParams.s2s = '1';
+      s2sParams.bidId = bidRequest.bidId;
+      s2sParams.bidWidth = playerWidth;
+      s2sParams.bidHeight = playerHeight;
+
+      if (bidderRequest && bidderRequest.gdprConsent) {
+        if (bidderRequest.gdprConsent.gdprApplies) {
+          AV_GDPR = 1;
```
Comment (idettman) [inline modules/aniviewBidAdapter.js]: GDPR data is being stored on the window object, is this intentional?
Here are the resulting window properties created from lines 100 and 101:
```
window.AV_GDPR = 1
window.AV_CONSENT = "..."
```

**Comment by itaynave** [comment]
@idettman thanks for your comment, Consent parameters fixed

**Pull request merged**